src/Inciter/Performer.C

```diff
@@ -375,7 +376,21 @@ Performer::updateSolution( const std::vector< std::size_t >& gid,
     // evaluateTime(), is called upon completion of the reduction.
     contribute(
       CkCallback( CkReductionTarget( Conductor, evaluateTime ), m_conductor ) );
-
+		// TEST FEATURE: Manually migrate this chare by using migrateMe to see if
```

Comment:
Let's keep this code in here but commented. We'll derive a regression test from his.